packages/audio-block/src/AudioBlock.tsx

```diff
@@ -22,19 +22,23 @@ import {
     hasRichTextValue,
     isDownloadable,
     joinClassNames,
+    withAttachmentsProvider,
 } from '@frontify/guideline-blocks-settings';
 import { useEffect, useState } from 'react';
-import 'tailwindcss/tailwind.css';
-import '@frontify/guideline-blocks-settings/styles';
+
 import { AudioPlayer, BlockAttachments, UploadPlaceholder } from './components';
 import { getDescriptionPlugins, titlePlugins } from './helpers/plugins';
-import { AUDIO_ID } from './settings';
+import { ATTACHMENTS_ASSET_ID, AUDIO_ID } from './settings';
 import { BlockSettings, TextPosition } from './types';
 
+import 'tailwindcss/tailwind.css';
+import '@frontify/guideline-blocks-settings/styles';
+import '@frontify/fondue/dist/styles.css';
```

Comment:
```suggestion
import '@frontify/fondue/style';
```

As mentioned here: https://github.com/Frontify/fondue

Reply:
fixed